docs/compiler.md

```diff
@@ -181,6 +181,64 @@ Deployment works via an RPC server, an address of which is passed via `-r`
 option and should be signed using a wallet from `-w` option. More details can
 be found in `deploy` command help.
 
+#### Config file
+Configuration file contains following options:
+
+| Parameter | Description | Example |
+| --- | --- | --- |
+| `name` | Contract name in the manifest. | `"My awesome contract"`
+| `safemethods` | List of methods which don't change contract state | `["balanceOf", "decimals"]`
+| `supportedstandards` | List of standards this contract implement. Current options: `NEP-11`, `NEP-17`, `NEP-11-Paylable`, `NEP-17-Payable`. This will enable additional checks in compiler. | `["NEP-17"]`
```

Comment:
s/`implement`/`implements`?

Reply:
I don't think it's correct to talk about "current options", any string is accepted and valid. There are some special standards, though, where NEP-11 and NEP-17 are real standars and "payable" things are some NeoGo optional extensions (I'm not sure they're worth mentioning because method-specific checks are enabled based on method presence and not on declared standard). `s,Paylable,Payable,` also.